templates/cluster-template-managed-ssh.yaml

```diff
@@ -29,7 +29,7 @@ spec:
   network: ${CLOUDSTACK_NETWORK_NAME}
   zone: ${CLOUDSTACK_ZONE_NAME}
 ---
-apiVersion: controlplane.cluster.x-k8s.io/v1alpha3
+apiVersion: controlplane.cluster.x-k8s.io/v1beta1
 kind: KubeadmControlPlane
```

Comment:
KubeAdmControlPlane changed slightly from v1alpha3 to v1alpha4.  See [Required cluster template changes](https://cluster-api.sigs.k8s.io/developer/providers/v0.3-to-v0.4.html#required-cluster-template-changes).  infrastructureTemplate becomes machineTemplate.infrastructureRef.

Reply:
Good catch! I need to get to where I actually use the templates directly so I notice these issues. Right now I've been maintaining my own version outside the repo. Will update shortly.